CustCartMethods.java

```diff
@@ -0,0 +1,142 @@
+import java.util.ArrayList;
+import java.util.List;
+/*
+ *Used to execute all the functions performed in a cart 
+ * 
+ */
+public class CustCartMethods
+{
+	public List<Product> cartvals=new ArrayList<Product>();
+	/**
+	 * Used to add items in the cart
+	 * @param pId represents product id
+	 * @param pQuantity represents product quantity
+	 * @return whether the product was added or not
+	 */
+	boolean add(int pId,int pQuantity) 

```

Comment:
use productId instead of pId. Variable name should be like any one who see the code will be able to understand it purpose quickly.